index.html

```diff
@@ -2914,11 +2912,11 @@ <h3>
           </pre>
 
           <p>
-  The contents of a data vault can also be searched to see if a certain attribute
-  name is indexed by using the <code>has</code> keyword.
+  The contents of a data vault can also be searched to see which documents have a certain attribute
+  key (regardless of attribute value) by using the <code>has</code> keyword.
           </p>
 
-          <pre class="example highlight" title="data vault query for a particular attribute name">
+          <pre class="example highlight" title="data vault query for a particular attribute key">
```

Comment:
The language above uses "attribute value" and "attribute name" -- wondering what we're going for wrt. consistency?

Reply:
Oops - I had intended to use the term "key" instead of "name" everywhere. Looks like I missed a few spots. I've updated it now.